scripts/chicken_dodo.lua

```diff
@@ -0,0 +1,120 @@
+include "constants.lua"
+
+local body, head, tail, lthigh, lknee, lshin, lfoot = piece('body', 'head', 'tail', 'lthigh', 'lknee', 'lshin', 'lfoot')
+local rthigh, rknee, rshin, rfoot, rsack, lsack, rblade = piece('rthigh', 'rknee', 'rshin', 'rfoot', 'rsack', 'lsack', 'rblade') 
+local lblade, spike1, spike2, spike3 = piece('lblade', 'spike1', 'spike2', 'spike3')
+
+-- signals --
+local SIG_MOVE = 2 -- original: 6, note that AIM and AIM2 are not used.
+
+
+local function WalkThread()
+	Signal(SIG_MOVE)
+	SetSignalMask(SIG_MOVE)
+	while true do
+		Turn(lthigh, x_axis, math.rad(70), math.rad(230))
+		Turn(lknee, x_axis, math.rad(-40), math.rad(270))
+		Turn(lshin, x_axis, math.rad(20), math.rad(270))
+		Turn(lfoot, x_axis, math.rad(-50), math.rad(420))
+		
+		Turn(rthigh, x_axis, math.rad(-20), math.rad(420))
+		Turn(rknee, x_axis, math.rad(-60), math.rad(420))
+		Turn(rshin, x_axis, math.rad(50), math.rad(420))
+		Turn(rfoot, x_axis, math.rad(30), math.rad(420))
+		
+		Turn(body, z_axis, math.rad(5), math.rad(40))
+		Turn(lthigh, z_axis, math.rad(-5), math.rad(40))
+		Turn(rthigh, z_axis, math.rad(-5), math.rad(40))
+		Move(body, y_axis, 0.7, 8000)
```

Comment:
Stick to the original (also the other 8000s)
```suggestion
		Move(body, y_axis, 0.7, 22)
```